helpers/test-helper.js

```diff
@@ -991,7 +992,12 @@ exports.getColonyEditable = async function getColonyEditable(colony, colonyNetwo
 exports.getWaitForNSubmissionsPromise = async function getWaitForNSubmissionsPromise(repCycleEthers, rootHash, nLeaves, jrh, n) {
   return new Promise(function (resolve, reject) {
     repCycleEthers.on("ReputationRootHashSubmitted", async (_miner, _hash, _nLeaves, _jrh, _entryIndex, event) => {
-      const nSubmissions = await repCycleEthers.getNSubmissionsForHash(rootHash, nLeaves, jrh);
+      let nSubmissions;
+      if (rootHash === _hash) {
```

Comment:
Can we add a comment explaining this behavior? I understand it now but some description of a scenario where `rootHash` would be different from `_hash` and why we handle it the way we do.